reference/ReferenceSeaport.sol

```diff
@@ -25,7 +25,7 @@ import { ReferenceOrderCombiner } from "./lib/ReferenceOrderCombiner.sol";
 import { OrderToExecute, AccumulatorStruct } from "./lib/ReferenceConsiderationStructs.sol";
 
 /**
- * @title ReferenceConsideration
+ * @title ReferenceSeaport
```

Comment:
This should be a new contract that imports ReferenceConsideration like on the optimized contracts so that we can override the name / name string